Update recipe based on template

aosthof opened the pull request.

aosthof committed:
Update recipe based on template

rsimai reviewed (changes requested):
The tables are broken and don't render correctly:

![image](https://user-images.githubusercontent.com/6737491/112654883-27ba9800-8e50-11eb-8de3-3001e4ba2f24.png)

aosthof committed:
Fix table rendering of roulades recipe

rsimai reviewed (changes requested):
the other table is also broken: 
![image](https://user-images.githubusercontent.com/6737491/112730478-15b12600-8f32-11eb-8fec-e9ac1862fd4d.png)

aosthof committed:
Update roulade_pretzel_dumplings.adoc

Fixed second table. Shouldn't have this done somewhere between diner, exchanging diapers and other stuff. ;)

rsimai reviewed (approved):
renders nicely and looks even leckerer :-)
